static/js/babelhook.js

```diff
@@ -2,8 +2,3 @@ const { babelSharedLoader } = require("../../webpack.config.shared");
 require('babel-polyfill');
 
 require('babel-register')(babelSharedLoader.query);
-
-// Force everything to the latest version of react
-let React = require('react');
-let mockRequire = require('mock-require');
```

Comment:
are we using `mock-require` anywhere else? if not it might be good to remove the dependency.